linebot/api.py

```diff
@@ -1125,6 +1123,260 @@ def get_insight_message_event(self, request_id, timeout=None):
 
         return InsightMessageEventResponse.new_from_json_dict(response.json)
 
+    def create_audience_group(self, audience_group_name, audiences, is_ifa=False, timeout=None):
```

Comment:
[FYI]
Due to [this update](https://developers.line.biz/en/reference/messaging-api/#request-body-12), so audiences should be changed to `optional`.

Reply:
It is fixed.